interface/Cargo.toml

```diff
@@ -47,3 +47,4 @@ frozen-abi = [
     "dep:solana-logger",
     "solana-pubkey/frozen-abi"
 ]
+test = ["solana-pubkey/std"]
```

Comment:
Consider adding `solana-system-interface = { path = ".", features = "test" }` to dev-dependencies.

If this was just added for `cargo hack` (I don't see it used anywhere else), you could instead do the following:
- remove this feature
- add `solana-pubkey = { version = "2.1", features = ["std"] }` to dev-dependencies
- add `--all-targets`  to the `cargo hack` command

Should probably add `--all-targets` either way